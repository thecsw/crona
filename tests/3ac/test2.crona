main : void() {
	a : byte;
	if (a > 9) {
		write 1;
	} else {
		write 0;
	}
}