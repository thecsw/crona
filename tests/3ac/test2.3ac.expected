[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN main LOCALS]
a (local var of 1 bytes)
tmp0 (tmp var of 1 bytes)
[END main LOCALS]
main:       enter main
            [tmp0] := [a] GT8 9
            IFZ [tmp0] GOTO lbl_1
            WRITE 1
            goto lbl_2
lbl_1:      nop
            WRITE 0
lbl_2:      nop
lbl_0:      leave main
